.github/workflows/support-packages.yml

```diff
@@ -0,0 +1,93 @@
+name: Build and push support-packages
+
+on:
+  workflow_call:
+    inputs:
+      branch-name:
+        type: string
+        required: true
+        description: Branch CI is running on
+
+jobs:
+  build-support-packages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Map collector to stackrox versions
+        run: |
+          ${{ github.workspace }}/kernel-modules/support-packages/01-collector-to-rox-version-map.py \
+            ${{ github.workspace }}/RELEASED_VERSIONS \
+            /tmp/support-packages/metadata
+
+      - name: Create driver version metadata
+        env:
+          BRANCH_NAME: ${{ inputs.branch-name }}
+        run: |
+          ./.github/support-packages/create-metadata-dir.sh
+
+      - name: Group versions by driver version
+        run: |
+          ${{ github.workspace }}/kernel-modules/support-packages/03-group-by-module-version.sh \
+            /tmp/support-packages/metadata
+
+      - name: Authenticate with GCP
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT }}'
+
+      - name: Create support-packages
+        run: |
+          # TODO: Change the bucket to the production one, once we are done testing
+          ${{ github.workspace }}/kernel-modules/support-packages/04-create-support-packages.sh \
+            ${{ github.workspace }}/collector/LICENSE-kernel-modules.txt \
+            gs://mauro-drivers-test/drivers \
+            /tmp/support-packages/metadata \
+            /tmp/support-packages/output
+
+      - name: Create index file
+        run: |
+          relative_path="collector/support-packages"
+
+          if [[ "${{ github.event_name == 'pull_request' }}" == "true" ]]; then
```

Comment:
So, those are useful once the workflow condition has been locally changed to run on a PR, right ?

Reply:
Yes, the idea is to not push support-packages to the production bucket when running PRs, since we could testing changes that break support packages.